src/app.rs

```diff
@@ -116,45 +118,35 @@ impl MacroSolverApp {
             bridge,
         }
     }
-}
 
-impl eframe::App for MacroSolverApp {
-    fn save(&mut self, storage: &mut dyn eframe::Storage) {
-        eframe::set_value(storage, "LOCALE", &self.locale);
-        eframe::set_value(storage, "RECIPE_CONFIG", &self.recipe_config);
-        eframe::set_value(storage, "SELECTED_FOOD", &self.selected_food);
-        eframe::set_value(storage, "SELECTED_POTION", &self.selected_potion);
-        eframe::set_value(storage, "CRAFTER_CONFIG", &self.crafter_config);
-        eframe::set_value(storage, "SOLVER_CONFIG", &self.solver_config);
-        eframe::set_value(storage, "MACRO_VIEW_CONFIG", &self.macro_view_config);
+    #[cfg(target_arch = "wasm32")]
+    fn initialize_bridge(cc: &eframe::CreationContext<'_>, data_update: &Rc<Cell<Option<SolverEvent>>>) -> BridgeType {
+        let ctx = cc.egui_ctx.clone();
+        let sender = data_update.clone();
 
-        eframe::set_value(storage, "CUSTOM_RECIPE", &self.custom_recipe);
-        eframe::set_value(storage, "RECIPE_SEARCH_TEXT", &self.recipe_search_text);
-        eframe::set_value(storage, "FOOD_SEARCH_TEXT", &self.food_search_text);
-        eframe::set_value(storage, "POTION_SEARCH_TEXT", &self.potion_search_text);
+        <Worker as gloo_worker::Spawnable>::spawner()
+            .callback(move |response| {
+                sender.set(Some(response));
+                ctx.request_repaint();
+            })
+            .spawn(concat!("./webworker", env!("RANDOM_SUFFIX"), ".js"))
     }
 
-    fn auto_save_interval(&self) -> std::time::Duration {
-        std::time::Duration::from_secs(1)
+    #[cfg(not(target_arch = "wasm32"))]
+    fn initialize_bridge(_cc: &eframe::CreationContext<'_>, _data_cell: &Rc<Cell<Option<SolverEvent>>>) -> BridgeType {
+        BridgeType::new()
     }
 
+    /// Called once before the first frame.
+    pub fn new(cc: &eframe::CreationContext<'_>) -> Self {
+        MacroSolverApp::setup_app(cc)
```

Comment:
Why the extra indirection and not just have the setup in `MacroSolverApp::new`?

Reply:
Fixed this, this is an artifact of a prior build